lib/SILOptimizer/UtilityPasses/SILSkippingChecker.cpp

```diff
@@ -51,6 +51,9 @@ static bool shouldHaveSkippedFunction(const SILFunction &F) {
   if (!func)
     return false;
 
+  if (func->getResilienceExpansion() == ResilienceExpansion::Minimal)
```

Comment:
It seems like the only condition that's needed here is F.isSerialized(). Everything else should follow, including minimal resilience expansion. If that doesn't there's an inconsistency elsewhere.